x/mongo/driver/ocsp/ocsp.go

```diff
@@ -52,8 +52,28 @@ func newOCSPError(wrapped error) error {
 	return &Error{wrapped: wrapped}
 }
 
+// Response contains a subset of the details needed from an OCSP response after the original response has been
+// validated.
+type Response struct {
```

Comment:
Fair point. The idea here was to keep around a subset of the response for caching purposes because we don't want to cache unnecessary information. Changed to `ResponseDetails`.